core/DataAccess/Model.php

```diff
@@ -349,7 +375,7 @@ public function getArchiveIdAndVisits($numericTable, $idSite, $period, $dateStar
                                OR name = '" . ArchiveSelector::NB_VISITS_CONVERTED_RECORD_LOOKED_UP . "')
                          $timeStampWhere
                          AND ts_archived IS NOT NULL
-                     ORDER BY ts_archived DESC, idarchive DESC";
+                     ORDER BY ts_archived ASC, idarchive DESC";
```

Comment:
shouldn't this still be DESC maybe?

Reply:
The way the rows are processed, the rows encountered latter overwrite the former in the result array (this is in ArchiveSelector.php). So using ASC means we end up using the latest archives. It was just easier to code than DESC where we'd have to stop when the ts_archived changed, but per name. Also, originally it was DESC so I could add a LIMIT, but because of how archives are structured w/ status flags & archive data together (and we need done flags + visits + conversions), there's no way to know how many rows we need to select.